workspace/template.ts

```diff
@@ -1,7 +1,73 @@
+
+const readline = require('readline');
+const request = require('request');
+
+import * as _ from "lodash"
+import {Bus} from './bus'
+import {Coordinates} from './coordinates'
+import {PostCode} from './postCode'
+import {BusStop} from './busStop'
+import {Result} from './result'
+
 export class Template {
```

Comment:
You can start renaming this, now you're properly taking ownership of what it does. It's not a template any more!